delatore/bot/main.py

```diff
@@ -85,23 +87,18 @@ def start(self):
 
 bot = DelatoreBot(BOT_CONFIG.token)
 
+STATUS = "status"
 
-@bot.message_handler(commands=["status"])
+
+@bot.message_handler(commands=[STATUS])
 def handle_start_help(message: Message):
-    message_text: str = message.text
-    LOGGER.debug("Got message: %s", message_text)
-    _, *arguments = message_text.split(" ")
-    if not arguments:
-        return
-    if len(arguments) > 1:
-        count = int(arguments[1])
-    else:
-        count = 1
-    target: list = arguments.pop(0).split(":")
-    template = None
-    if target[0] == "awx":
-        if len(target) > 1:
-            template = target[1]
-        LOGGER.debug("Getting AWX status")
-        response = AWXApiClient().create_status_message(template)
+    try:
+        source, template_name, count = parse_command(message.text)
+    except CommandParsingError:
+        response = bot.reply_to(message, "Invalid command. Please check command syntax")
+        return bot.alert(message.chat.id, response)
+    try:
+        response = AWXApiClient().create_status_message(template_name)
         bot.alert(message.chat.id, response)
+    except:
```

Comment:
Exception should be raised inside of `create_status_message` and be more precise. `except` is catching all exceptions